core/src/main/java/edu/wpi/grip/core/Main.java

```diff
@@ -1,6 +1,38 @@
 package edu.wpi.grip.core;
 
+import com.google.common.eventbus.EventBus;
+import edu.wpi.grip.core.operations.Operations;
+import edu.wpi.grip.core.serialization.Project;
+import edu.wpi.grip.generated.CVOperations;
+
+import java.io.File;
+
+/**
+ * Main driver class for headless mode
+ */
 public class Main {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws Exception {
+        if (args.length != 1) {
+            System.err.println("Usage: GRIP.jar project.grip");
+            return;
+        }
+
+        final String projectPath = args[0];
+
+        final EventBus eventBus = new EventBus();
```

Comment:
This should at least print the stack trace. The built in logger for the event bus sucks.